code/modules/mob/living/bloodcrawl.dm

```diff
@@ -1,30 +1,3 @@
-/obj/effect/dummy/phased_mob/slaughter //Can't use the wizard one, blocked by jaunt/slow
```

Comment:
There was a comment about this.

Reply:
There is be no reason to use the wizard jaunt path anyway unless you want it to work like it.

Reply:
What i'm saying you put the nojaunt turf check on base type. Either mention this in the changelog at least or keep the old functionality.

Reply:
I'll mention it in the changelog. That check exists to stop mobs from phasing where they shouldn't such as hilbert hotel, multicam and transit space.